install.sh

```diff
@@ -3,73 +3,116 @@
 # RITA is brought to you by Offensive CounterMeasures.
 # offensivecountermeasures.com
 
+_NAME=$(basename "${0}")
+_FAILED="\e[91mFAILED\e[0m"
+_SUCCESS="\e[92mSUCCESS\e[0m"
+
+#Error handling
+#Kill 0 to kill subshells as well
+trap "printf '\n[!] Installation $_FAILED!\n'; kill 0" ERR INT
 set -o errexit
+set -o errtrace
 set -o pipefail
 
-_NAME=$(basename "${0}")
-_INSDIR="/usr/local"
+# Make sure to source the latest .bashrc
+# Hack the PS1 variable to get around ubuntu .bashrc
+OLD_PS1=$PS1
+PS1=" "
+# Hack the interactive flag to get around other .bashrc's
+set -i
+
+source $HOME/.bashrc
```

Comment:
I'm having the same issue as @joelillo. My script is erroring out on this line. I'll post more updates if I can track down the cause.

Reply:
Weird. Do you have a .bashrc file? I'll have to do some testing to narrow this down. I remember it worked for me at the time :/  The whole dance seems to be in place just to source the bashrc file automatically (i.e. if the user runs the install script twice in a row, we want to have the most up to date info on the status of the install).

Most bash rc files prevent being sourced from an interactive session. Ubuntu uses PS1 to prevent this, other systems check the interactive flag.

Reply:
There is a .bashrc file, but it's just the default one that came with the OS.  I removed it and the script fails for a different reason.  But if I make an empty .bashrc then it goes through the install fine (until it fails later on due to the different branches).